powerdns/models/requests.py

```diff
@@ -25,6 +25,38 @@
 log = logging.getLogger(__name__)
 
 
+def can_auto_accept_record_request(user_request, user, action):
+    """
+    Return bool if `user_request` (RecordRequest or Record DeleteRequest) being
```

Comment:
Btw could you describe briefly possible cases here? What happens when user is owner of domain/record?